classes/01class/exercises/c01-git02/albertoeks/ANSWER.md

```diff
@@ -0,0 +1,22 @@
+# c01-git02
+
+## Answers
+
+1. Is it possible to see the inner commit of squashed commit with `git rebase`?
+```
+Yes, it is. Through git reflog command we should be able to see the squashed commits.
```

Comment:
Good, and even when you don't have the reflog anymore, if you have the commit ID you can check it out.